wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -320,6 +325,9 @@ public void initialize(TransformContext context) throws Exception {
       );
     }
 
+    boolean keepOld = Boolean.valueOf(context.getArguments().get(KEEP_OLD_DEFAULT_KEY));
```

Comment:
nit: I know this probably doesn't cause issues, but I would like to have an explicit null check here before invoking the valueOf method.

Reply:
Can we have string `wrangler.error.strategy.default` instead? It would be more flexible and look as a feature, not as a patch. This way one can set any default he/she wants, not "old" or "new"